src/plasmapy/simulation/particle_tracker/particle_tracker.py

```diff
@@ -133,6 +133,45 @@ class to the ``run`` method.
     If the particle's position is not ``NaN``, but the velocity is ``NaN``, the particle has been stopped
     (i.e. it is still in the simulation but is no longer evolved.)
     If both the particle's position and velocity are set to ``NaN``, then the particle has been removed from the simulation.
+
+    Example
+    -----
+    >>> from plasmapy.particles import CustomParticle
+    >>> from plasmapy.plasma.grids import CartesianGrid
```

Comment:
I think a more common use case would be using `Particle` to create an actual particle, like `Particle('p+')` A 1 kg, 1 C particle isn't a very common scenario!

Reply:
That's a good point! I made have made and committed that change.